core/src/trezor/lvglui/scrs/template.py

```
@@ -2245,6 +2263,44 @@ def __init__(
             )
 
 
+class TronVoteWitness(FullSizeWindow):
+    def __init__(
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)